testing/test_aux.out

```diff
@@ -77,6 +83,9 @@ port-02 module_config modules
   "brute": {
     "enabled": true
   },
+  "discover": {
```

Comment:
bizareley appearing twice

Reply:
https://github.com/arupiot/daq/blob/124084d4fdfb2773e8d6600f8f7b9d24d62319ec/testing/test_aux.out

It appears twice like all the other modules appear twice - those are the configs for 2 ports